code/modules/surgery/robotics.dm

```diff
@@ -230,7 +230,7 @@
 					"You start mending the damage to [target]'s [I.name]'s mechanisms." )
 
 	target.custom_pain("The pain in your [affected.name] is living hell!", 75)
-	..()
+	..()  
```

Comment:
```suggestion
	..()
```
Your leftover spaces continue to haunt you

Reply:
rofl